src/SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests/StringExtensions.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests
+{
+    public static class StringExtensions
+    {
+        public static Guid? ToGuid(this string value)
+        {
+            Guid? id = null;
+            if (!string.IsNullOrWhiteSpace(value))
+            {
+                id = Guid.Parse(value);
+            }
```

Comment:
I'm not sure this makes a lot of sense.  We are guarding against a blank input, but not against malformed input - e.g. `i_am_not_a_guid`.

It would make sense to me to either say translate null input to null output and fail on invalid input, or translate all invalid input to null output.  Translating _some_ invalid input to null and failing on others seems not very obvious.